etcdserver/v3demo_server.go

```diff
@@ -103,16 +105,23 @@ func (s *EtcdServer) Watchable() dstorage.Watchable {
 	return s.kv
 }
 
+const (
+	// noTxn is an invalid txn ID.
+	// To apply with independent Range, Put, Delete, you can pass noTxn
+	// to apply functions instead of a valid txn ID.
+	noTxn = -1
```

Comment:
it does not unless i was wrong. TxnBegin should only return positive number as ID (well an ID should be positive).